src/compiler/builderState.ts

```diff
@@ -264,7 +264,9 @@ namespace ts {
          */
         export function create(newProgram: Program, getCanonicalFileName: GetCanonicalFileName, oldState?: Readonly<BuilderState>, disableUseFileVersionAsSignature?: boolean): BuilderState {
             const fileInfos = new Map<Path, FileInfo>();
-            const referencedMap = newProgram.getCompilerOptions().module !== ModuleKind.None ? createManyToManyPathMap() : undefined;
+            const options = newProgram.getCompilerOptions();

```

Comment:
I don't understand the consequences of this change - if there's no multimap, what does it do?  Just use the first file?

Reply:
Any changed file needs the whole program emit so there is no use of having the references map since it’s unnecessary work

Reply:
So, to confirm, setting the multi-map to undefined means "emit everything"?

Reply:
Yes. Module None means global change, so emit everything and similarly --out is same.

Reply:
Thanks!